estate/static/src/js/tour.js

```diff
@@ -0,0 +1,10 @@
+import { registry } from "@web/core/registry";
+
+
+registry.category("web_tour.tours").add('estate_tour', {
+    url: "/web",
```

Comment:
url is now defined in the xml web_tour (and should point to /odoo). But since this is the default value, you don't have to :-)
```suggestion
```